Detections/MultipleDataSources/AuditPolicyManipulation_using_auditpol.yaml

```diff
@@ -0,0 +1,90 @@
+  | parse CommandLine with * "/subcategory:" subcategorytoken
+  | extend SubCategory = tostring(split(subcategorytoken, "\"")[1]) , Toggle =  tostring(split(subcategorytoken, "\"")[2])
+  | where SubCategory in~ (SubCategoryList) //use in~ for inclusion or !in~ for exclusion
+  | where Toggle !in~ ("/failure:disable", " /success:enable /failure:disable") // use this filter if required to exclude certain toggles
+  | project TimeGenerated, Computer, Account, SubjectDomainName,  SubjectUserName, Process, ParentProcessName,  CommandLine, SubCategory, Toggle
+  | extend timestamp = TimeGenerated, AccountCustomEntity = Account, HostCustomEntity = Computer
+  ),
+  (
+  DeviceProcessEvents
+  | where TimeGenerated >= ago(timeframe)
+  // | where InitiatingProcessFileName =~ "auditpol.exe" 
+  | where InitiatingProcessCommandLine has_any (tokens)
+  | where AccountName !in~ (AccountAllowList)
+  | parse InitiatingProcessCommandLine with * "/subcategory:" subcategorytoken
+  | extend SubCategory = tostring(split(subcategorytoken, "\"")[1]) , Toggle =  tostring(split(subcategorytoken, "\"")[2])
+  | where SubCategory in~ (SubCategoryList) //use in~ for inclusion or !in~ for exclusion
+  | where Toggle !in~ ("/failure:disable", " /success:enable /failure:disable") // use this filter if required to exclude certain toggles
+  | project TimeGenerated, DeviceName, AccountName, InitiatingProcessAccountDomain, InitiatingProcessAccountName, InitiatingProcessFileName, InitiatingProcessParentFileName,  InitiatingProcessCommandLine, SubCategory, Toggle
+  | extend timestamp = TimeGenerated, AccountCustomEntity = AccountName, HostCustomEntity = DeviceName
+  ),
+  (
+  Event
+  | where TimeGenerated > ago(timeframe)
+  | where Source == "Microsoft-Windows-Sysmon"
+  | where EventID == 1
+  | extend EventData = parse_xml(EventData).DataItem.EventData.Data
+  | mv-expand bagexpansion=array EventData
+  | evaluate bag_unpack(EventData)
+  | extend Key=tostring(['@Name']), Value=['#text']
+  | evaluate pivot(Key, any(Value), TimeGenerated, Source, EventLog, Computer, EventLevel, EventLevelName, EventID, UserName, RenderedDescription, MG, ManagementGroupName, Type, _ResourceId)
+  // | where OriginalFileName =~ "auditpol.exe"
+  | where CommandLine has_any (tokens)
+  | where User !in~ (AccountAllowList)
+  | parse CommandLine with * "/subcategory:" subcategorytoken
+  | extend SubCategory = tostring(split(subcategorytoken, "\"")[1]) , Toggle =  tostring(split(subcategorytoken, "\"")[2])
+  | where SubCategory in~ (SubCategoryList) //use in~ for inclusion or !in~ for exclusion
+  | where Toggle !in~ ("/failure:disable", " /success:enable /failure:disable") // use this filter if required to exclude certain toggles
+  | project TimeGenerated, Computer, User, Process, ParentImage,  CommandLine, SubCategory, Toggle
+  | extend timestamp = TimeGenerated, AccountCustomEntity = User, HostCustomEntity = Computer
```

Comment:
Let's make sure Type is brought thru since we are using union on several different tables.